src/mito2/src/engine/tests.rs

```diff
@@ -287,11 +288,100 @@ async fn test_write_to_region() {
     assert_eq!(num_rows, rows_inserted);
 }
 
+#[tokio::test]
+async fn test_region_replay() {
+    common_telemetry::init_default_ut_logging();
+    let mut env = TestEnv::with_prefix("region-replay");
+    let engine = env.create_engine(MitoConfig::default()).await;
+
+    let region_id = RegionId::new(1, 1);
+    let request = CreateRequestBuilder::new().build();
+    let region_dir = request.region_dir.clone();
+
+    let column_schemas = request
+        .column_metadatas
+        .iter()
+        .map(column_metadata_to_column_schema)
+        .collect::<Vec<_>>();
+    engine
+        .handle_request(region_id, RegionRequest::Create(request))
+        .await
+        .unwrap();
+
+    let rows = Rows {
+        schema: column_schemas.clone(),
+        rows: build_rows(0, 20),
+    };
+    let output = engine
+        .handle_request(region_id, RegionRequest::Put(RegionPutRequest { rows }))
+        .await
+        .unwrap();
+    let Output::AffectedRows(rows_inserted) = output else {
+        unreachable!()
+    };
+    assert_eq!(20, rows_inserted);
```

Comment:
We already test output in write tests, we don't need to assert it in all remaining tests.